src/app/expense/expense-list/expense-list.component.scss

```diff
@@ -0,0 +1,49 @@
+
+//Table design
+table {
+    border-collapse: collapse;
+    width: 100%;
+    max-width: 800px;
+    margin: 0 auto;
+    font-family: Arial, sans-serif;
+    font-size: 16px;
+  }
+  //Table heading and data design
+  th, td {
+    border: 1px solid #ccc;
+    padding: 10px;
+    text-align: center;
+    
+  }
+  //Table heading design
+  th {
+    background-color: #f2f2f2;
+  }
+  
+  tr:nth-child(even) {
+    background-color: #f8f8f8;
+  }
+  
+  tr:hover {
+    background-color: #e8e8e8;
+  }
+  //For table mode
+  @media only screen and (max-width: 767px) {
+   table{
+    max-width: 600px;
+    font-size: 14px;
+   }
+   th td{
+    padding: 5px;
+   }
+}
+//For mobile mode
+   @media only screen and (max-width: 480px) {
+   table{
+    max-width: 400px;
+    font-size: 12px;
+   }
+   th td{
+    padding: 3px;
+   }
+  }
```

Comment:
In this file replace all color code and font-size with variables, this applies to all files

Reply:
You have declared variables in component, please declare variables in common scss file and then re-use it so you don't have to declare same variables in each component